linear_operator/operators/_linear_operator.py

```diff
@@ -6,11 +6,18 @@
 import math
 import numbers
 import warnings
-from abc import ABC, abstractmethod
+from abc import abstractmethod
 from copy import deepcopy
-from typing import Any, Callable, Dict, Optional, Tuple, Union
+from typing import Any, Callable, Dict, Iterable, Optional, Tuple, Union
 
+import numpy as np
 import torch
+
+try:
+    # optional library for advanced type signatures
+    from jaxtyping import Float
```

Comment:
Since jaxtyping is going to be a non-optional dependency, we shouldn't need this try/except block.

Reply:
Agreed.